### PR TITLE
Fix radix dropdown z index

### PR DESCRIPTION
**Pull request by luthfib**
Fix z-index with radix dropdown menu. Change background color of condition 2.0 to white, to better reflect design.

### CODE REVIEW

**Review by pmadria3 — DISMISSED** [review]
LGTM

File: src/components/core/main.scss [hunk 1]
```diff
@@ -124,4 +124,9 @@
   .ctw-alert-bg {
     @apply ctw-flex ctw-items-start ctw-space-x-2 ctw-p-4 ctw-text-left;
   }
+
+  .ctw-provider [data-radix-popper-content-wrapper] {
```
Comment (WesleyKapow) [inline src/components/core/main.scss:128]: Blocker: As discussed, we should target direct descendant which I believe is `.ctw-provider > [data-radix-popper-content-wrapper]`, that way we won't target any other ones of these if they happened to exist (via consuming app using radix as well).

**Review by WesleyKapow — CHANGES_REQUESTED** [review]
One blocker, otherwise lgtm

File: src/components/core/main.scss (hunk 1 above)
Comment (WesleyKapow) [reply src/components/core/main.scss]: https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator

**Review by WesleyKapow — APPROVED** [review]

**Pull request merged**